github3/github.py

```diff
@@ -279,10 +279,17 @@ def check_authorization(self, access_token):
             return self._boolean(resp, 200, 404)
         return False
 
+    @requires_auth
     def create_gist(self, description, files, public=True):
         """Create a new gist.
 
-        If no login was provided, it will be anonymous.
+        .. versionchanged:: 1.1.0
+
+        Per `GitHub's recent announcement`_ authentication is now required for
```

Comment:
These two lines need to be indented to be part of the versionchanged directive.

Reply:
Updated!